lang/lang_en_nl.txt

```diff
@@ -376,7 +376,7 @@
 
 #MSG_FILAMENT_CLEAN c=20 r=2
 "Filament extruding & with correct color?"
-"Filament extrudeerd + en de juiste kleur?"
+"Filament extrudeert en met de juiste kleur?"
```

Comment:
Too long as it is limited to 20 characters and 2 rows
"Filament extrudeert met de juiste kleur?"